### DISCUSSION TITLE
Find error on uv slope

### DISCUSSION THREAD

**Discussion by JudeVijayanga** [mboquien/cigale#108] [General]
I want to find the error on uv slope. Cigale doesn't give error on uv slope. Can you suggest a method to do this? If I able to run cigale multiple times considering photometry variation with normal distribution, I can find. But I don't how to do this.

**Comment by mboquien** [comment]
The error in the UV slope should be given by `bayes.param.beta_calz94_err`. For this you need to enable the computation of the UV slope (set `beta_calz94` to `True` in the `restframe_parameters` module) and add it to the list of parameters to be analyzed in the `variable` line of the `pdf_analysis` module.

**Reply by JudeVijayanga** [reply]
Thank you professor.  I will try. Can I do the same thing for  luminosity of galex.fuv?

**Reply by mboquien** [reply]
Yes. In the list of bands in the `pdf_analysis module`, you can list `galex.fuv`.

**Reply by JudeVijayanga** [reply]
Thank you very much. Seems like it's working.
